pithos/plugins/mpris.py

```diff
@@ -424,6 +420,20 @@ def _get_metadata(self, window, song):
 
         return metadata
 
+    def _metadata_equal(self, m1, m2):
```

Comment:
Shouldn't direct comparisons of dicts work just fine?, if GVariant doesn't implement comparisons that should probably be fixed upstream too.